cluster-up/cluster/kind/common.sh

```diff
@@ -141,6 +141,35 @@ function _fix_node_labels() {
     done
 }
 
+function _get_cri_bridge_mtu() {
+  docker network inspect -f '{{index .Options "com.docker.network.driver.mtu"}}' bridge
+}
+
+function _patch_calico_manifest() {
+  local -r calico_manifest="$1"
+  local -r calico_diff="$2"
+  local -r destination_dir="$3"
+
+  local -r cri_mtu=$(_get_cri_bridge_mtu)
+  local -r ipip_mode=$(cat "$calico_manifest" | grep CALICO_IPV4POOL_IPIP -A 1 | grep -Po "(Always|Never)")
```

Comment:
Using `sed` capturing groups you don't need piping.

Reply:
Done